wiki/en/Installation-for-Linux.md

```diff
@@ -54,24 +54,45 @@ For installers on other distributions, see their package managers and [Repology]
 
 ## Set up your hardware
 
+The Jamulus client needs to connect to a running [JACK](https://jackaudio.org/) server in order to start. You need to find out which is the low-latency sound server of your distribution.
+If your distribution uses JACK, check out how to [configure JACK with QjackCtl](Installation-for-Linux#configure-jack-with-qjackctl), on the other hand, if it is PipeWire, check out the section called [configure PipeWire](Installation-for-Linux#configure-pipewire) instead.
```

Comment:
```suggestion
If your distribution uses JACK itself, check out how to [configure JACK with QjackCtl](Installation-for-Linux#configure-jack-with-qjackctl).

If your distribution uses PipeWire as a JACK server, check out the section called [configure PipeWire](Installation-for-Linux#configure-pipewire) instead.
```

Reply:
I don't know if people who needs this article understand what is jackd and/or jackdbus...
That's why I think "JACK" as to be prefered.

Reply:
done